plasmapy/formulary/shocks.py

```diff
@@ -0,0 +1,122 @@
+""" This module gathers functions relating to shocks and the properties thereof.
+"""
+__all__ = ["entropy_jump_polytropic"]
+
+
+from astropy import units as u
+from astropy.constants import c, a0, k_B
+from numpy import pi, exp, sqrt, log, nan, isclose
+from warnings import warn
+
+from plasmapy.utils.exceptions import PhysicsWarning
+from plasmapy.utils.decorators import (
+    angular_freq_to_hz,
+    check_relativistic,
+    validate_quantities,
+)
+
+
+@validate_quantities(
+    rho_1={"can_be_negative": False, "pass_equivalent_units": True},
+    rho_2={"can_be_negative": False, "pass_equivalent_units": True},
+    p_1={"can_be_negative": True, "pass_equivalent_units": True},
+    p_2={"can_be_negative": True, "pass_equivalent_units": True},
+    c_v={"can_be_negative": False},
+)
+def entropy_jump_polytropic(
+    c_v: u.J / u.K,
+    p_1: u.Pa,
+    p_2: u.Pa,
+    rho_1: u.kg / u.m ** 3,
+    rho_2: u.kg / u.m ** 3,
+    gamma=5 / 3,
+) -> u.J / u.K:
+    r"""
+    Entropy is not conserved across a shock, since a shock is
+    an irreversible, non-adiabatic process. This equation is based on the specific entropy
+    of a polytropic gas before and after the shock has passed. The shock
```

Comment:
It wouldn't hurt to define polytropic here or somewhere (since this documentation is meant to be educational)

Reply:
This is a good point.  You can probably get way with something like...

```
    Entropy is not conserved across a shock, since a shock is
    an irreversible, non-adiabatic process. This equation is based on the specific
    entropy of a polytropic gas ( :math:`p \propto \rho^\gamma`) before and after
    the shock has passed. The shock is an isolated steady, perpendicular shock,
    flowing through a non magnetized, polytropic gas.
```